MVCCore.Mediator.Request/LoginRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using MVCCore.Models;
+using MediatR;
+
+
+namespace MVCCore.Mediator.Request
+{
```

Comment:
Duplicate files, this is also there in handler

Reply:
Only single file with this name , No duplicacy